code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -63,6 +63,7 @@
 
 /mob/living/simple_animal/bot/secbot/beepsky/armsky
 	name = "Sergeant-At-Armsky"
+	desc = "It's Sergeant-At-Armsky! A disgruntled assistant to the Warden. He would probably shoot you if he had hands."
```

Comment:
when you examine something, isn't the first line already "its [thingy]!"

Reply:
It's current state
![image](https://user-images.githubusercontent.com/53777086/176653507-3d21aab3-065b-4e35-a474-c8bcd518c197.png)

Reply:
Yeah, that's right. Before the edit he had no description line, and was drawing from the top most variant of Beepsky, which happens to be Beep O'sky.